### DISCUSSION TITLE
57주차 반성문

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#600] [6. 반성문]
요즘 이래저래 정신이 없어서 글을 쓸 여유를 내기가 어려웠네요,,,
해외봉사단 준비에 졸업과제, 세차새차, 센터 업무에 더해 오랫동안 못봤던 친구들과 밀린 약속도 잡고 바쁘게 지냈습니다. 방학인데 평소랑 똑같이 바쁜 것 같아요.
프론트엔드 공부에도 시간을 내고 싶은데 일을 너무 많이 벌려놓은 것 같습니다. 업보빔을 그대로 맞고 있어요........
감당 가능한 정도로 일을 벌려야하는데 여간 쉽지가 않군요

**Comment by shkisme** [comment]
헤르미온느도 아마란스 스케줄 보면 기겁하고 도망갈거같아요
